tools/pipelines/templates/include-set-package-version.yml

```diff
@@ -44,8 +60,10 @@ steps:
       echo TEST_BUILD=$TEST_BUILD
       echo VERSION_RELEASE=$VERSION_RELEASE
       echo VERSION_PATCH=$VERSION_PATCH
+      echo VERSION_INCLUDE_INTERNAL_VERSIONS=$VERSION_INCLUDE_INTERNAL_VERSIONS
 
-      fluid-build-version
+      flub --help
```

Comment:
Is this intentional? Just to display some info about the script before running it?

Reply:
Yeah. I'll add a comment.